projects/core/tokens/notification-options.ts

```diff
@@ -1,29 +1,45 @@
-import {InjectionToken} from '@angular/core';
+import {InjectionToken, ValueProvider} from '@angular/core';
 import {TuiContextWithImplicit} from '@taiga-ui/cdk';
 import {TuiNotification} from '@taiga-ui/core/enums';
 import {TuiNotificationAutoClose} from '@taiga-ui/core/interfaces';
 import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
 
-export interface NotificationTokenOptions {
+export interface TuiNotificationDefaultOptions {
     readonly autoClose: TuiNotificationAutoClose;
     readonly label: PolymorpheusContent<TuiContextWithImplicit<TuiNotification>>;
     readonly status: TuiNotification;
     readonly hasIcon: boolean;
     readonly hasCloseButton: boolean;
+    readonly defaultAutoCloseTime: number;
 }
 
+/**
+ * @deprecated: remove in v3.0, use TuiNotificationDefaultOptions
+ */
+export type NotificationTokenOptions = TuiNotificationDefaultOptions;
```

Comment:
Do we need word `default` ?

I mean
```suggestion
export type NotificationTokenOptions = TuiNotificationOptions;
```

Other similar options dont have it in their naming.
Example
https://github.com/Tinkoff/taiga-ui/blob/bd5c1fbd7b8b6672e22a459bf13ef9d34f7923ba/projects/addon-commerce/components/money/money-options.ts#L6

Reply:
I have problem with existing interface

https://github.com/Tinkoff/taiga-ui/blob/main/projects%2Fcore%2Finterfaces%2Fnotification-options.ts